cmake/PackageConfig.cmake.in

```diff
@@ -2,10 +2,15 @@
 
 #  * @PROJECT_NAME@::@PROJECT_NAME@_4 - real32 library target
 #  * @PROJECT_NAME@::@PROJECT_NAME@_8 - real64 library target
+#  * @PROJECT_NAME@::@PROJECT_NAME@_d - mixed precision library target
 
 # Include targets file.  This will create IMPORTED target @PROJECT_NAME@
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
 
+if(@OpenMP_Fortran_FOUND@)
```

Comment:
Can you retain the comment line above this if-block.  Otherwise, when this is resolved it will look like this:
```
if(ON)
  find_dependency(OpenMP)
endif()
```

Reply:
replacing  `if(@OpenMP_Fortran_FOUND@)` with `if(@OPENMP@)` works.